lib/engine/train.rb

```diff
@@ -9,10 +9,10 @@ class Train
 
     attr_accessor :obsolete, :operated, :events, :variants, :obsolete_on, :rusted, :rusts_on, :index, :name,
                   :distance, :reserved
-    attr_reader :available_on, :discount, :multiplier, :sym, :variant
+    attr_reader :available_on, :discount, :multiplier, :sym, :variant, :requires_token
     attr_writer :buyable
 
-    def initialize(name:, distance:, price:, index: 0, **opts)
+    def initialize(name:, distance:, price:, index: 0, requires_token: true, **opts)
```

Comment:
just put it in opts

Reply:
I originally had it there, but Steve advised this made setting the default true much cleaner. I can move it back, though.

Reply:
yea let's move it back